testutils/src/main/java/com/amplifyframework/testutils/sync/SynchronousStorage.java

```diff
@@ -38,25 +40,33 @@
  * code, where we want to make a series of sequential assertions after
  * performing various operations.
  */
-@SuppressWarnings("unused")
 public final class SynchronousStorage {
-    // 5 seconds seemed to be insufficient to reliably cover both initial auth calls + storage network ops
-    private static final long STORAGE_OPERATION_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(10);
+    private static final long STORAGE_OPERATION_TIMEOUT_MS = TimeUnit.SECONDS.toMillis(5);
```

Comment:
Hmm, I guess having smaller default value is fine since `SynchronousStorage` does allow timeout override for especially long operations (e.g. large file upload)